webpack/webpack.config.js

```diff
@@ -123,7 +124,7 @@ module.exports = {
           {
             loader: "file-loader",
             options: {
-              name: "./[hash]-[name].[ext]"
+              name: "./assets/[hash]-[name].[ext]"
```

Comment:
for some reason this doesnt catch `sprite.svg` file 🤔

Reply:
yes, look at `svg-compiler-plugin.js` 😩 I managed to move it into `assets` by setting `compilation.assets["assets/sprite.svg"]` but then UI wouldn't show the icons.

Reply:
ah, thanks! we also have to change `./src/js/index.js@L39` to load the sprite from assets… 🤷‍♂️

Reply:
It didn't resolve the issue for me

Reply:
hm, what exactly are you testing? 🤔

Reply:
I do the build and then test against `http-server`. I check if all the assets are in `/dist/assets` and that the UI still has all the styles, images and icons.

Reply:
gotcha, it tries to load images from `http://127.0.0.1:4200/assets/assets/…` 😞